dipy/core/tests/test_sphere.py

```diff
@@ -357,5 +358,28 @@ def test_disperse_charges():
     nt.assert_array_almost_equal(norms, 1)
 
 
+def test_disperse_charges_alt():
+    # Create a random set of points
+    charges = np.random.random((3, 3))
+    charges /= vector_norm(charges, keepdims=True)
+    init_pointset = HemiSphere(xyz=charges).vertices
```

Comment:
Is this needed?

Reply:
Indeed, it was not. It was just making all vectors dwell in the `>0` octant actually. Thanks again.